dev/com.ibm.ws.webcontainer/resources/com/ibm/ws/webcontainer/resources/Messages.nlsprops

```diff
@@ -1603,3 +1603,18 @@ exception.occurred.while.running.ServletContainerInitializers.onStartup.useracti
 no.servlet.defined.for.servlet.filter.name.mapping=CWWWC0002W: No servlet definition is found for the {0} servlet name in the {1} filter mapping.
 no.servlet.defined.for.servlet.filter.name.mapping.explanation=The servlet name is used in filter mapping, but its definition has not been defined.
 no.servlet.defined.for.servlet.filter.name.mapping.useraction=Define the servlet which is used in the servlet name filter mapping.
+
+# {0} Cookie name that has conflict SameSite attribute
+# {1} current SameSite attribute value
+# {2} new SameSite attribute value
+duplicate.samesite.attribute.for.cookie.name=CWWWC0003W: A {0} cookie name with the existing SameSite attribute value {1} is configured with the {2} new value. No SameSite attribute is added for this cookie.
+duplicate.samesite.attribute.for.cookie.name.explanation=A cookie name cannot have multiple SameSite attribute values. The attribute is not added for the cookie.
+duplicate.samesite.attribute.for.cookie.name.useraction=The deployment descriptor's context parameters cookie names for 'SameSiteCookies_Lax', 'SameSiteCookies_None', and 'SameSiteCookies_Strict' must be unique across all three lists. 
+
+duplicate.reported.for.samesite.attribute.cookie.name=CWWWC0004W: The {1} SameSite attribute value already exists for the {0} cookie name. No SameSite attribute is added for this cookie.
+duplicate.reported.for.samesite.attribute.cookie.name.explanation=A cookie name cannot have multiple SameSite attribute values. The attribute is not added for the cookie.
+duplicate.reported.for.samesite.attribute.cookie.name.useraction=The cookie name set in the deployment descriptor's context parameters for 'SameSiteCookies_Lax', 'SameSiteCookies_None', and 'SameSiteCookies_Strict' must be unique across all three lists. 
```

Comment:
**change**
The cookie name set in the deployment descriptor's context parameters for 'SameSiteCookies_Lax', 'SameSiteCookies_None', and 'SameSiteCookies_Strict' must be unique across all three lists.
**to**
Check the deployment descriptor configuration to make sure the cookie name is configured for only one of the following SameSite attribute values: 'SameSiteCookies_Lax', 'SameSiteCookies_None', or 'SameSiteCookies_Strict'.